post/release-0-2.qmd

```diff
@@ -0,0 +1,353 @@
+
+The latest data is available from the [geoarrow.org data page](https://geoarrow.org/data)
+and the sources can be found at the
+[geoarrow/geoarrow-data](https://github.com/geoarrow/geoarrow-data) GitHub repository.
+
+## Ecosystem
+
+### GeoPandas
+
+In the landmark [GeoPandas](https://geopandas.org) 1.0 release, the `GeoSeries` and `GeoDataFrame`
+added `from_arrow()` and `to_arrow()` methods to import and export Python objects using the
+[Arrow PyCapsule Interface](https://arrow.apache.org/docs/format/CDataInterface/PyCapsuleInterface.html).
+
+```python
+import pyarrow as pa
+import geoarrow.pyarrow as ga
+from geoarrow.rust.io import read_flatgeobuf
+import geopandas
+
+url = "https://raw.githubusercontent.com/geoarrow/geoarrow-data/v0.2.0-rc6/natural-earth/files/natural-earth_cities.fgb"
+
+table_rs = read_flatgeobuf(url)
+df = geopandas.GeoDataFrame.from_arrow(table_rs)
+df.head(5)
+#>            name                   geometry
+#> 0  Vatican City  POINT (12.45339 41.90328)
+#> 1    San Marino   POINT (12.44177 43.9361)
+#> 2         Vaduz   POINT (9.51667 47.13372)
+#> 3       Lobamba     POINT (31.2 -26.46667)
+#> 4    Luxembourg      POINT (6.13 49.61166)
+
+pa.table(df.to_arrow())
+#> pyarrow.Table
+#> name: string
+#> geometry: extension<geoarrow.wkb<WkbType>>
+#> ----
+#> name: [["Vatican City","San Marino","Vaduz","Lobamba","Luxembourg",...,
+#> geometry: [[010100000054E57B4622E828408B074AC09EF34440,0101000000DC...
+```
+
+For more information, see the excellent GeoPandas documentation:
+
+- [`GeoSeries.from_arrow()`](https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoSeries.from_arrow.html)
+- [`GeoSeries.to_arrow()`](https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoSeries.to_arrow.html)
+- [`GeoDataFrame.from_arrow()`](https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoDataFrame.from_arrow.html)
+- [`GeoDataFrame.to_arrow()`](https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoDataFrame.to_arrow.html)
+
+Thanks to [@jorisvandenbossche](https://github.com/jorisvandenbossche) and
+[@kylebarron](https://github.com/kylebarron), for driving this integration!
+
+### Lonboard
+
+What's a fast geospatial skateboard? A *lon*board, of course! Powered by
+[deck.gl](https://deck.gl/), [lonboard](https://github.com/developmentseed/lonboard)
+can effortlessly render millions of features on an interactive map. The rendering
+is powered by [GeoArrow's native encodings](https://geoarrow.org/format.html) and
+the [GeoArrow extension types](https://geoarrow.org/extension-types.html) power
+geometry column detection and on-the-fly reprojection to ensure multiple layers are
+plotted together properly. The `viz()` one-liner works on any
+[Arrow-ish Python object](https://arrow.apache.org/docs/format/CDataInterface/PyCapsuleInterface.html)
+and is a massive quality-of-life improvement when working with large-ish data in a notebook.
+
+```python
+from geoarrow.rust.io import read_flatgeobuf
```

Comment:
I'd rather not suggest geoarrow.rust.io here, just yet 😅. 

Perhaps we could suggest `pyogrio`? That might also be a good cross-ecosystem example.